python/ray/serve/multiplex.py

```diff
@@ -72,13 +119,19 @@ async def load_model(self, model_id: str) -> Any:
                 and len(self.models) >= self.max_num_models_per_replica
             ):
                 # Unload the least recently used model.
+                self.models_unload_counter.inc()
+                unload_start_time = time.time()
                 await self.unload_model()
+                self.model_unload_latency_s.set(time.time() - unload_start_time)
             # Load the model.
             logger.info(f"Loading model '{model_id}'.")
+            load_start_time = time.time()
             if self.self_arg is None:
                 self.models[model_id] = await self._func(model_id)
             else:
                 self.models[model_id] = await self._func(self.self_arg, model_id)
+            self._push_multiplexed_replica_info = True
```

Comment:
Do you have to set `_push_multiplexed_replica_info` to True in `unload_models`?

Reply:
When there is cache miss, we will load new models and set the _push_multiplexed_replica_info to True.
Unload model is the intermediate step for loading new models.